pkg/steps/release/create_release.go

```diff
@@ -203,8 +203,17 @@ func (s *assembleReleaseStep) Run(ctx context.Context, dry bool) error {
 		}
 		return fmt.Errorf("could not resolve imagestream %s: %v", streamName, err)
 	}
-	cvo, ok := resolvePullSpec(stable, "cluster-version-operator", true)
-	if !ok {
+	var cvo string
+	if err := wait.Poll(10*time.Second, 5*time.Minute, func() (bool, error) {
+		pullSpec, ok := resolvePullSpec(stable, "cluster-version-operator", true)
+		if !ok {
+			log.Printf("waiting for importing cluster-version-operator ...")
```

Comment:
We don't need to print this (potentially) 30 times in the logs -- once before we start is fine

Reply:
Done.